### PR TITLE
Presign/base styles

### PR DESCRIPTION
**Pull request by fairlighteth**
# Summary

- Improve status label styles
- Dark mode only > Detect Gnosis Safe icon > invert color + grayscale
- Pill enum refactor
- Other misc styles refactor

https://user-images.githubusercontent.com/31534717/136819796-ff85d00e-b2c8-40a4-9d9f-eb001495ec86.mov

https://user-images.githubusercontent.com/31534717/136819829-418e3f1a-e0af-4b7b-b41c-6aea09a0da26.mov

https://user-images.githubusercontent.com/31534717/136819841-cf463ec1-b672-46e4-a1c9-98e867350b07.mov

Screen Shot 2021-10-11 at 17 47 27
Screen Shot 2021-10-11 at 17 47 20
Screen Shot 2021-10-11 at 17 47 08
Screen Shot 2021-10-11 at 17 46 59
Screen Shot 2021-10-11 at 17 46 50

### CODE REVIEW

File: src/custom/abis/types/factories/GPv2Settlement__factory.ts [hunk 1]
```diff
@@ -2,86 +2,92 @@
 /* tslint:disable */
```
Comment (biocom) [inline src/custom/abis/types/factories/GPv2Settlement__factory.ts:2]: linter?

File: tsconfig.json [hunk 2]
```diff
@@ -2,7 +2,11 @@
   "extends": "./src/custom/tsconfig-paths-custom.json",
```
Comment (biocom) [inline tsconfig.json:2]: linter?

**Review by anxolin — APPROVED** [review]

**Review by W3stside — APPROVED** [review]

File: tsconfig.json (hunk 2 above)
Comment (alfetopito) [reply tsconfig.json]: Yep.
For some reason, local formatting always applies one format, CI applies another.

Have to fix this, eventually...

**Review by alfetopito — APPROVED** [review]

**Comment by elena-zh** [comment]
Moreover, I see this message in the activity log when I have already sighed the transaction (1 signer is in the Safe policy)
Screenshot_2

**Comment by biocom** [comment]
@elena-zh Thanks for reviewing again. Great we have separate tickets for some items.

> Wrap transaction log/modal/toast notification

Yes we definitely need to fix this.

> I'd write 'Approving' instead of the 'Approve' in the waiting modal

In my opinion 'Approving' means an action is enforce/being executed, whereas 'Approve' is the action you're about to execute/sign off. The latter seems more accurate in this scenario?

> Besides, in the latest PRs it has become impossible to close a toast notification in a mobile view. Let me know please it I need to create a separate issue for this

Please, if you could create a separate ticket for this, that would be great. This way we can keep the PR going. Btw same for the Safe/threshold comments you mentioned. I think it's worth it to have a ticket in order for us to review this.

**Comment by biocom** [comment]
@elena-zh Yes that makes sense!

Btw, another thing we need as a todo is this modal
Screen Shot 2021-10-13 at 18 08 48

This is shown when I want to cancel an order. You see the text in the second step isn't correct. We have logic to address this, but it's not getting a prop yet I think, which would define the 'type' of tx/order (reference: `src/custom/components/TransactionConfirmationModal/index.tsx` L435)

**Comment by elena-zh** [comment]
@biocom , yes.. I even haven't noticed this 2nd step here 😭
I have created #1610 issue for this

**Comment by elena-zh** [comment]
Hi @biocom , I have finished with testing Safe app transactions. Changes LGTM here, however, I'd add a margin between warning and the link to view transactions. Let me know please whether it is fixed in another pr or I need to open a separate issue for this case
add margin

**Pull request merged**